arch/arm/src/armv7-a/l2cc.h

```diff
@@ -70,6 +70,22 @@ extern "C"
 void arm_l2ccinitialize(void);
 #endif
 
+/****************************************************************************
+ * Name: l2cc_get_linesize
+ *
+ * Description:
+ *    Get L2CC-P310 L2 cache linesize
+ *
+ * Input Parameters:
+ *    None
+ *
+ * Returned Value:
+ *    L2 cache linesize
+ *
+ ****************************************************************************/
+
+uint32_t l2cc_get_linesize(void);
```

Comment:
Maybe this can be a define instead of function?

Reply:
l2cc_get_linesize() is a function in l2cc.h, but the MARCO is in l2cc_pl310.h,
l2cc.h is hard to directly include l2cc_pl310.h

Reply:
Why? Based on the description it is L2CC-P310 specific.

Reply:
l2cc_pl310.h is just one type of l2cc.
And l2cc.h is a common header file, l2cc_pl310.h is one of realize of l2cc.

Reply:
So maybe than let's update the description from `Get L2CC-P310 L2 cache linesize` to `Get L2CC L2 cache linesize`?

Reply:
done